src/main/java/streams/Main.java

```diff
@@ -0,0 +1,51 @@
+package streams;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        String[] keywords = {"abstract", "assert", "boolean", "break", "byte", "case",
+                "catch", "char", "class", "const", "continue", "default", "do", "double",
+                "else", "enum", "extends", "final", "finally", "float", "for", "if",
+                "implements", "import", "instanceof", "int", "interface", "long", "native",
+                "new", "package", "private", "protected", "public", "return", "short",
+                "static", "strictfp", "super", "switch", "synchronized", "this", "throw",
+                "throws", "transient", "try", "void", "volatile", "while"};
+        int[] keywordCount = new int[keywords.length];
```

Comment:
Instead of using parallel arrays, use Map<String, Integer> keywords object. This enhances readability and simplifies the code by associating each keyword directly with its count.